sdk/websites/Azure.ResourceManager.AppService/src/Customization/Models/AppServiceTableStorageApplicationLogsConfig.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ComponentModel;
+using Azure.Core;
+
+namespace Azure.ResourceManager.AppService.Models
+{
+    public partial class AppServiceTableStorageApplicationLogsConfig
+    {
+        /// <summary> Uri of the resource. </summary>
+        [EditorBrowsableAttribute(EditorBrowsableState.Never)]
+        [ObsoleteAttribute("This property has been replaced by ResourceUriString", false)]
```

Comment:
When we use an attribute, we usually do not include the `Attribute` suffix. Let us remove the suffix here